server/opts.go

```diff
@@ -4131,12 +4131,20 @@ func parseAuthorization(v any, errors *[]error) (*authorization, error) {
 		case "token":
 			auth.token = mv.(string)
 		case "timeout":
-			at := float64(1)
+			at := float64(0)
```

Comment:
Use `0` as default, so that in `setBaselineOptions()` the [call](https://github.com/nats-io/nats-server/blob/8f8fe68d4c7d9ec72fa33181a4b376eb02a095ec/server/opts.go#L5455) to `getDefaultAuthTimeout` will work as expected.